cshub-client/yarn.lock

```diff
@@ -7776,14 +7696,16 @@ vuetify@^1.3.5:
   integrity sha512-HnMBBtO5FqTPzr6FXVlxnYulGMrHVlcmBL7SSLE/AQnNbdUpT6Jw9D/Kdw9fbK5mdn7JSWMcfWpPJUrqYTtK1w==
 
 vuex-typex@^3.0.1:
-  version "3.1.2"
-  resolved "https://registry.yarnpkg.com/vuex-typex/-/vuex-typex-3.1.2.tgz#9f60f52f92185cec4875910108d37a838648333b"
-  integrity sha512-kKMvPKf68u/P6ILBFxhcqrvJwNDdboJ4dCl+IazbgvQaVb2pKHLGDu+7oYC/vZnbCMv3aQrCOPN8saB49GPxAQ==
+  version "3.0.1"
+  resolved "https://registry.yarnpkg.com/vuex-typex/-/vuex-typex-3.0.1.tgz#f54619d76081e2bd902120d145284a3ba0a95758"
+  integrity sha1-9UYZ12CB4r2QISDRRShKO6CpV1g=
+  dependencies:
+    vuex "^3.0.0"
 
-vuex@^3.0.1:
-  version "3.1.0"
-  resolved "https://registry.yarnpkg.com/vuex/-/vuex-3.1.0.tgz#634b81515cf0cfe976bd1ffe9601755e51f843b9"
-  integrity sha512-mdHeHT/7u4BncpUZMlxNaIdcN/HIt1GsGG5LKByArvYG/v6DvHcOxvDCts+7SRdCoIRGllK8IMZvQtQXLppDYg==
+vuex@^3.0.0, vuex@^3.0.1:
```

Comment:
I think the rest doesn't really matter, but this might have something to do with it: vuex-typex now getting an older version of vuex